src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -234,10 +239,11 @@ const SubmissionDetails = _.flow(
               size: { basis: 125, grow: 0 },
               headerRenderer: () => h(Sortable, { sort, field: 'cost', onSort: setSort }, ['Run Cost']),
               cellRenderer: ({ rowIndex }) => {
-                return cost ? h(TextCell, [Utils.formatUSD(filteredWorkflows[rowIndex].cost)]) : 'N/A'
+                // handle undefined workflow cost as $0
+                return cost ? h(TextCell, [Utils.formatUSD(filteredWorkflows[rowIndex].cost || 0)]) : 'N/A'
```

Comment:
Do we still want the ternary here?

Reply:
I think so. If `cost` hasn't loaded for the workflow yet, I think it's better to show nothing than state that it cost nothing?

Reply:
Ah, right, I was confusing myself.